docs/developer-docs/latest/guides/auth-request.md

```diff
@@ -116,13 +145,19 @@ The API response contains the **user's JWT** in the `jwt` key.
 }
 ```
 
-You will have to store this `JWT` in your application, it's important because you will have to use it the next requests.
+Save the `JWT` in your application or copy it to your clipboard. You will use it to make future requests. 
+
+::: NOTE
+See the [login documentation](/developer-docs/latest/plugins/users-permissions.md#login) for more information.
+:::
 
 ### Fetch articles
 
-Let's fetch Articles you created.
+Fetch the Articles you created earlier by sending a **GET** request to the `/articles` route:
 
-To do so, you will have to fetch `/articles` route in **GET**.
+:::: tabs card
```

Comment:
Are you planning to add the Postman tab here as well?

Reply:
A screenshot of the GET request in Postman would be best IMO. There is no **body**, so the user essentially just adds the URL to the field and clicks "Send".
![GET-request-unauth](https://user-images.githubusercontent.com/60552605/185442754-b894f878-a729-4814-b4b9-aa504c230994.png)